packages/components/TestimonialSection.tsx

```diff
@@ -19,6 +26,72 @@ export default function TestimonialSection() {
       >
         {t(`title`)}
       </Heading>
+      {testimonialsList.map((testimonialData) => {
+        return (
+          <TestimonialCard
+            key={testimonialData.name}
+            name={testimonialData.name}
+            testimonial={testimonialData.testimonial}
+            img={testimonialData.img}
+            borderColor={testimonialData.borderColor}
+          />
+        );
+      })}
     </Section>
   );
 }
+
+interface TestimonialCardProps {
+  name: string;
+  testimonial: string;
+  img: string;
+  borderColor: string;
+}
+
+function TestimonialCard({
+  name,
+  testimonial,
+  img,
+  borderColor,
+}: TestimonialCardProps) {
+  return (
+    <Box maxW="400px" maxH="400" rounded="lg" shadow="lg" p={2}>
+      <Flex justifyContent="space-between" mb="1">
+        <Heading alignSelf="center" size="sm">
+          {name}
+        </Heading>
+        <Image
+          src={img}
+          alt={`img${name}`}
+          maxW={{ base: '60px', sm: '70px' }}
+          fit="cover"
+          rounded="full"
+          borderStyle="solid"
+          borderWidth="3px"
+          borderColor={borderColor}
+        />
+      </Flex>
+      <Text
+        maxH="300px"
+        overflow="auto"
+        css={{
+          '&::-webkit-scrollbar': {
+            width: '0px',
```

Comment:
this css hides scrollbar that may appear due to overflow property (line 76)

Reply:
what if we use overflow hidden?

Reply:
it shows a ugly scrollbar on the side of the <Text/>

Reply:
then you wont be able to see the rest of the text

Reply:
this is the behaviour now (=auto)
![chrome-capture-2022-5-7](https://user-images.githubusercontent.com/86134825/172495593-4abdeacb-a8d7-4598-9bd0-25a08924df0b.gif)
> if =hidden then you cant interact with the text

Reply:
usando o inspect aq me parece tranquilo

![chrome-capture-2022-5-14](https://user-images.githubusercontent.com/86134825/173715892-6f3b4ec7-5a5f-4138-9368-650d5d34b57d.gif)

Reply:
tentei outras propriedades de overflow mas nenhuma deu resultado similar

Reply:
```suggestion
        css={{
          '&::-webkit-scrollbar': {
            display: 'none',
```
dessa forma da tmb!

Reply:
Eh, não sei se a gente deveria esconder o scroll viu. Isso eh um antipattern

Reply:
Na real, o ideal eh não ter texto tão grande que precise de scrollbar

Reply:
Limitando o tamanho do texto a gente consegue evitar isso

Reply:
https://stackoverflow.com/questions/18716863/css-hide-scroll-bar-if-not-needed tentou esse `overflow-y` já?

Reply:
outro link interessante
https://betterprogramming.pub/a-guide-to-using-the-css-overflow-property-5fd5bbd038c9